--- gcc-core.ori/gcc/toplev.c	2015-11-28 20:23:21.503801295 +0100
+++ gcc-core/gcc/toplev.c	2015-11-28 20:24:19.272242918 +0100
@@ -531,7 +531,7 @@
    for floor_log2 and exact_log2; see toplev.h.  That construct, however,
    conflicts with the ISO C++ One Definition Rule.   */
 
-#if GCC_VERSION < 3004 || !defined (__cplusplus)
+/* #if GCC_VERSION < 3004 || !defined (__cplusplus) */
 
 /* Given X, an unsigned number, return the largest int Y such that 2**Y <= X.
    If X is 0, return -1.  */
@@ -583,7 +583,7 @@
 #endif
 }
 
-#endif /*  GCC_VERSION < 3004 || !defined (__cplusplus)  */
+/* #endif /\*  GCC_VERSION < 3004 || !defined (__cplusplus)  *\/ */
 
 /* Handler for fatal signals, such as SIGSEGV.  These are transformed
    into ICE messages, which is much more user friendly.  In case the
